src/json-patch-ot-agent.js

```diff
@@ -72,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
```

Comment:
That's too late. This only happens if base queue decides to apply the patch.
That's just how the protocol works - at first the initial state gets sent, so we should reset accordingly